data/assets/examples/scale/nonuniformstaticscale/ellipsoid.asset

```diff
@@ -0,0 +1,29 @@
+-- This asset creates a SceneGraphNode that is rendering a sphere which is adjust to an
+-- ellipsoidal shape by using a non-uniform scaling. In particular, the second axis of
+-- symmetry is half as long as the first, and the third axis of symmetry is a third as
```

Comment:
I'm a little confused what the axes of symmetries are - is there a way to rephrase this? First, second, and third axis? x, y, and z?

Reply:
Fine to rename it to just "axes", but from Wikipedia:
> An ellipsoid has three pairwise [perpendicular](https://en.wikipedia.org/wiki/Perpendicular) [axes of symmetry](https://en.wikipedia.org/wiki/Rotational_symmetry) which intersect at a [center of symmetry](https://en.wikipedia.org/wiki/Central_symmetry), called the center of the ellipsoid